pants.toml

```diff
@@ -98,6 +98,36 @@ pants_ignore.add = [
   "!/pants.pex",
 ]
 
+[source]
+
+roots = [
```

Comment:
Although the help docs make it clear, reading just this text makes it sound like we have a source root at src/cpp and we don't. Maybe `root_patterns`?

Reply:
+1. I really like `root_patterns`.

Reply:
I can do that, and we can even continue to support `*` wildcards, I guess.

Reply:
Yeah, that will also make the `pants.toml` entry much shorter.